src/phases/egg-lapse-phase.ts

```diff
@@ -1,11 +1,18 @@
 import BattleScene from "#app/battle-scene.js";
-import { Egg } from "#app/data/egg.js";
+import { Egg, EGG_SEED } from "#app/data/egg.js";
 import { Phase } from "#app/phase.js";
 import i18next from "i18next";
 import Overrides from "#app/overrides";
 import { EggHatchPhase } from "./egg-hatch-phase";
+import { Mode } from "#app/ui/ui.js";
+import { achvs } from "#app/system/achv.js";
+import { PlayerPokemon } from "#app/field/pokemon.js";
+import { EggSummaryPhase } from "./egg-summary-phase";
+import { EggHatchData } from "#app/data/egg-hatch-data.js";
```

Comment:
```suggestion
import { Mode } from "#app/ui/ui";
import { achvs } from "#app/system/achv";
import { PlayerPokemon } from "#app/field/pokemon";
import { EggSummaryPhase } from "./egg-summary-phase";
import { EggHatchData } from "#app/data/egg-hatch-data";
```
NIT: Same comment as above